modules/gumgumBidAdapter.js

```diff
@@ -3,6 +3,7 @@ import * as utils from '../src/utils.js'
 import { BANNER, VIDEO } from '../src/mediaTypes.js';
 
 import { config } from '../src/config.js'
+import { createEidObject } from './userId/eids.js'
```

Comment:
Instead of modifying pbjs, try using: `createEidsArray` from `eids.js`.  
It's already exported and I see it's being used by a lot of other bidders already.